src/components/one-time-donation/components/ThirdStep.tsx

```diff
@@ -0,0 +1,137 @@
+import { Collapse, Grid, List, Divider, Typography, Button } from '@mui/material'
+import { createStyles, makeStyles } from '@mui/styles'
+import RadioGroupFormik from './RadioGroupFormik'
+import { useField } from 'formik'
+import { UseQueryResult } from 'react-query'
+import { useRouter } from 'next/router'
+import { routes } from 'common/routes'
+import { CampaignResponse } from 'gql/campaigns'
+import { useViewCampaign } from 'common/hooks/campaigns'
+import { CopyTextButton } from '../../common/CopyTextButton'
+import { useTranslation } from 'next-i18next'
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    h2: {
+      fontFamily: 'Montserrat',
+      fontStyle: 'normal',
+      fontWeight: '500',
+      fontSize: '35px',
+      lineHeight: '120%',
+      marginTop: '58px',
+      marginBottom: '59px',
+      letterSpacing: '-0.5px',
+      color: '#343434',
+    },
+    divider: {
+      border: '1px solid #000000',
+    },
+    uncheked: {
+      width: '284px',
+      height: '75px',
+      background: '#FFFFFF',
+      border: '1px solid #000000',
+      boxSizing: 'border-box',
+      borderRadius: '37.5px',
+      marginBottom: '33px',
+    },
+    checked: {
+      width: '284px',
+      height: '75px',
+      background: '#D2F0FF',
+      border: '1px solid #000000',
+      boxSizing: 'border-box',
+      borderRadius: '37.5px',
+      marginBottom: '33px',
+    },
+    body: {
+      maxWidth: '657px',
+      marginLeft: 'auto',
+      marginRight: 'auto',
+    },
+  }),
+)
+
+export default function ThirdStep() {
+  const classes = useStyles()
+  const [field] = useField('payment')
+  const router = useRouter()
+  const slug = String(router.query.slug)
+  const { data }: UseQueryResult<{ campaign: CampaignResponse }> = useViewCampaign(slug as string)
+  const { t } = useTranslation('one-time-donation')
+  const bankAccountInfo = {
+    owner: t('third-step.owner'),
+    bank: t('third-step.bank'),
+    iban: 'BG66 UNCR 7000 1524 3490 32',
```

Comment:
Please reuse the constant `ibanNumber` from https://github.com/podkrepi-bg/frontend/blob/1ba1b816db44ee1ebe7eb0fa39d5c850c6879496/src/common/iban.ts#L1